src/osm_configurator/view/utilityframes/export_frame.py

```diff
@@ -32,6 +30,6 @@ def freeze(self):
 
     def unfreeze(self):
         """
-        If this method is called, the frame returns into its previous interactable state.
+        If this method is called, the frame returns into its previous intractable state.
```

Comment:
tbh doens't matter this class will be deleted anyway

Reply:
ok